test/index.js

```diff
@@ -102,6 +102,7 @@ test('time out when there is no response', function (t) {
   })
   setTimeout(function () {
     t.equal(typeof error, 'object')
+    service.timeOut = 1000
```

Comment:
just resetting the timeout on a FeatureService instance that is shared throughout the tests